### DISCUSSION TITLE
You are forbidden from performing this action on github.com after linking ID to join a classroom

### DISCUSSION THREAD

**Discussion by MaddyGuthridge** [community/community#33572] [GitHub Education]
When I tried to join a classroom, after linking my ID, I got an error 502. Now when I load the classrooms page, I can't see any classes, even though my account has been linked according to the class admin panel. This issue is affecting over 70% of the 500 students in my cohort. How can it be fixed so we can start our work?

**Comment by snowden-fu** [comment]
lgtm

**Comment by snowden-fu** [comment]
suggest note on title it is for COMP9900 of UNSW @MiguelGuthridge
